### PR TITLE
fix: progressive profiling should be full height

### PR DESCRIPTION
**Pull request by ahtesham-quraish**
Description:

### Description

- Progressive profiling form should be full height

#### How Has This Been Tested?

 - it has been tested locally

#### Screenshots:

|After|
|-----|
|image|

### CODE REVIEW

File: src/authn-component/index.jsx [hunk 1]
```diff
@@ -130,6 +130,7 @@ export const AuthnComponent = ({
       close={close}
       hasCloseButton={hasCloseButton}
       size={screenSize}
+      isFullScreen={screenSize === 'fullscreen'}
```
Comment (mubbsharanwar) [inline src/authn-component/index.jsx:133]: Why do we need to pass this prop, We are already passing `size` prop and we can add this condition inside the `BaseContainer` component.

File: src/base-container/index.jsx [hunk 2]
```diff
@@ -37,7 +39,12 @@ const BaseContainer = ({
       isBlocking
       variant="default"
       title="authn-component"
-      className="bg-light-200 authn-component__modal"
+      className={classNames(
+        'bg-light-200 authn-component__modal',
+        {
+          'authn-component__modal-full-height': isFullScreen,
```
Comment (mubbsharanwar) [inline src/base-container/index.jsx:45]: Like:
```suggestion
          'authn-component__modal-full-height': size === 'fullscreen',
```
And by doing this, we don't need to pass extra prop.

File: src/authn-component/index.jsx (hunk 1 above)
Comment (ahtesham-quraish) [reply src/authn-component/index.jsx:133]: yes you are right we can do that

File: src/base-container/index.scss [hunk 3]
```diff
@@ -1,7 +1,11 @@
 @import "~@edx/brand-edx.org/paragon/variables";
 
 .authn-component__modal {
-  max-height: 98vh !important;
+  max-height: 98vh;
```
Comment (mubbsharanwar) [inline src/base-container/index.scss:4]: Would this be applied in the prospectus without important? We need to confirm it.

**Review by mubbsharanwar — APPROVED** [review]

**Pull request merged**